Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1201,14 +1201,25 @@ public string GetSmartApiUrl(string hostname, int? port = null)
         }
 
         /// <inheritdoc/>
-        public string GetLoopbackHttpApiUrl()
+        public string GetInterfaceHttpApiUrl()
```

Comment:
`Interface` is a bit ambiguous.

Reply:
GetUrlForUseByHttpApi ??

Reply:
`GetLoopbackHttpApiUrl` isn't that far from incorrect now. Maybe `GetLocalHttpApiUrl` or `GetLanHttpApiUrl`?